3-enrich/emr-etl-runner/Gemfile

```diff
@@ -26,6 +26,7 @@ gem "awrence", "~> 0.1.0"
 gem "snowplow-tracker", "~> 0.5.2"
 gem "iglu-ruby-client", ">= 0.1.0"
 gem "aws-sdk-s3", "~> 1"
+gem "aws-sdk-emr", "~> 1.1.0"
```

Comment:
Elasticity also has the necessary function https://github.com/rslifka/elasticity/blob/master/lib/elasticity/emr.rb#L124-L135.

I generally agree that we should prefer the official SDK rather than wrapper, but in sake of consistency I'd not introduce new dependencies.

Reply:
Implemented using Elasticity EMR class instead.